spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/java/com/alibaba/cloud/nacos/ribbon/NacosRule.java

```diff
@@ -96,4 +125,53 @@ public Server choose(Object key) {
 	public void initWithNiwsConfig(IClientConfig iClientConfig) {
 	}
 
+	private String getIPv4() {
```

Comment:
Whether we can storage IP address when the app register to registry? and then we can use the address directly? But there is a problem that when the app don't register, we should to search the specific IP address again. BTW, the name of method, we can modify to getAppLocalIPv4Address? same below.

Reply:
‘Whether we can storage IP address when the app register to registry? and then we can use the address directly? ’ The value of IPv4 and IPv6 has been storaged when init,there is a methed named initIp.